.eslintrc

```diff
@@ -43,36 +50,35 @@ rules:
   prefer-spread: 2
   prefer-template: 2
 
-globals:
-  # Meteor globals
-  Meteor: false
-  DDP: false
-  Mongo: false
-  Session: false
-  Accounts: false
-  Template: false
-  Blaze: false
-  UI: false
-  Match: false
-  check: false
-  Tracker: false
-  Deps: false
-  ReactiveVar: false
-  EJSON: false
-  HTTP: false
-  Email: false
-  Assets: false
-  Handlebars: false
-  Package: false
-  App: false
-  Npm: false
-  Tinytest: false
-  Random: false
-  HTML: false
+  # eslint-plugin-meteor
+  meteor/globals: 2
+  meteor/no-zero-timeout: 2
+  meteor/no-session: 0
+  meteor/pubsub: 2
+  meteor/core: 2
+  meteor/methods: 2
+  meteor/check: 2
+  meteor/connections: 2
+  meteor/collections: 2
+  meteor/session: [2, 'no-equal']
+
+settings:
+  meteor:
 
+    # Our collections
+    collections:
```

Comment:
By telling ESLint-plugin-Meteor about the names of the collections, they will automatically be registered as globals.

Furthermore using `collection.find()` etc will then be linted. And assigning something other than `new Mongo.Collection()` to a collection will error out, because most likely you are doing something wrong then; although there are valid cases like in this project.